libs/scripts/scripts.js

```diff
@@ -103,6 +103,7 @@ const locales = {
 const config = {
   geoRouting: 'off',
   fallbackRouting: 'off',
+  links: 'on',
```

Comment:
the options are supposed to be default options that are sometimes overwritten by metadata. Metadata boolean values are usually "on" or "off". Since these options are supposed to be default options for what can be set in the metadata, we opted to give them the same values that they would have in the metadata. 
Having said all that, I do think it's a great point. config.links is technically also not overridable currently with metadata, like geoRouting and fallbackRouting are. So I could give it a boolean value. It just will look inconsistent with the other options.